### PR TITLE
docs: Add discontinue statement for this project

### PR DESCRIPTION
**Pull request by Xuanwo**
Fix https://github.com/alibaba/AliSQL/issues/112

Cc @xpchild 

This PR will also:

- Fix https://github.com/alibaba/AliSQL/issues/111
- Fix https://github.com/alibaba/AliSQL/issues/107
- Fix https://github.com/alibaba/AliSQL/issues/102
- Fix https://github.com/alibaba/AliSQL/issues/100
- Fix https://github.com/alibaba/AliSQL/issues/98
- Fix https://github.com/alibaba/AliSQL/issues/96
- Fix https://github.com/alibaba/AliSQL/issues/92
- Fix https://github.com/alibaba/AliSQL/issues/90

---

Reasons for discontinuing:

> https://github.com/alibaba/AliSQL/issues/112#issuecomment-1239034070
>
> 对于 5.6 的版本：1. 官方已经停止维护 2. 大量客户还在 5.6 版本上使用 3. 历史原因导致 8.0 没有在同一个repo上。
导致我们没有在 5.6上做更新，但又不能关闭。

### CODE REVIEW

File: README.md
Comment (faketygg) [inline README.md:5]: galaxyengine and alisql are not maintained by one team, and is also not actively maintained, so how about using stonedb instead?

```suggestion
**AliSQL has been discontinued, please use [stoneatom/stonedb](https://github.com/stoneatom/stonedb) instead.**
```

Comment (Xuanwo) [reply README.md:5]: @xpchild Add a link `Repository AliSQL 8.0` to https://github.com/ApsaraDB/galaxyengine in commit https://github.com/alibaba/AliSQL/commit/25a2aefb4329a62db747e74611888cbb4fff87a4, so I use it directly.

I do not know about its internal maintenance relation. Any repo LGTM if maintainer approved.

**Review by wey-gu — APPROVED** [review]
LGTM, thanks!

**Comment by BohuTANG** [comment]
Former contributor of AliSQL 6 years ago, I suggest setting this repo to archive, cc @xpchild

**Comment by Xuanwo** [comment]
An empty response is the best discontinue statement, I got it.

**Pull request closed without merging**
